### DISCUSSION TITLE
ETHER NO GET

### DISCUSSION THREAD

**Discussion by mSalimrKhan** [smartcontractkit/full-blockchain-solidity-course-py#605] [Q&A] [answered]
facing problenm after poisting in tweet Rinkeby Authenticated Faucet but stil not get ether

**Comment by cromewar** [comment, marked as answer]
The rinkeby Faucet is down, you should use the chainlink faucet instead, or post your wallet address here I'll send you some eth.

**Comment by mSalimrKhan** [comment]
Thnx for replyin my adress is 0xB32De7B4df7333989482824af9068028e7bC6719

**Reply by cromewar** [reply]
Sent: https://rinkeby.etherscan.io/tx/0x1c6890cd9b5b51bf1a48a9ccb18ebb6d9115db0fc98ca0e4e0cfa3310ff9ca25
